peering/peering.go

```diff
@@ -220,6 +220,7 @@ func (ps *PeeringService) AddPeer(info peer.AddrInfo) {
 		handler.setAddrs(info.Addrs)
 	} else {
 		logger.Infow("peer added", "peer", info.ID, "addrs", info.Addrs)
+		// in case ps.host is null this line results in a panic.
```

Comment:
If host is nil, pretty much nothing will work. There's no need to leave a comment to that effect.

Reply:
Fixed.